lib/ImagePrototypeInit.js

```diff
@@ -47,6 +47,19 @@
         this.__locked = false;
     };
 
+    Image.prototype.getMetadata = function() {
+        if (!this.__metadata || this.__metadata === "") {
```

Comment:
That's probably a good idea, I changed it so that users can set metadata to an empty string (tEXt chunk with empty comment field) or null (i.e. no tEXt chunks).